### ISSUE TITLE
[Enhancement] Allow for divider lines in declarative list 

### ISSUE THREAD

**Issue by MadsBuchmann** [kirbydesign/designsystem#1909]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
The current template driven list allows for lines to be drawn between items by setting the input property `[showDividers]="true"`. 
It can be seen on [the cookbook showcase page for list](https://cookbook.kirby.design/#/home/showcase/list) under the headline  "List with dividers". 

The new declarative list should also allow for this in some way.

## Describe the solution you'd like
I do not know exactly how we should do this. 

Following the doctrine of _HTML > CSS > Javascript_ I think we could look into just applying a `with-dividers` class or similar to either the separate items or the containing list (or perhaps allow for both 🙀?).

## Have you considered any alternatives?
Allow it to be set as an input property on the `kirby-list` element as is done in the template driven list. But it seems a overly complex to me, to use an input property for something that can just be achieved by using a class.

## Are there any additional context?
On the branch `poc/1813-show-divider-declarative-list` some work relating to this has been stashed away. We initially began working on this as part of #1813; but decided to create a separate issue for this instead.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by MadsBuchmann** [comment]
@RasmusKjeldgaard - still relevant, righty?

**Comment by RasmusKjeldgaard** [comment]
> @RasmusKjeldgaard - still relevant, righty?

I believe so!
